### DISCUSSION TITLE
Combo field?

### DISCUSSION THREAD

**Discussion by cstrat** [mantinedev/mantine#120] [Q&A] [answered]
Any thoughts/progress on a `combo` type field for forms?

It wouldn't be too difficult to hack together, I am happy to do it but would need help converting it to typescript.

Using the base `textinput` element and a combination of the overlay/menu that comes with the tag picker to provide a filter/suggested list of options.

**Comment by rtivital** [comment, marked as answer]
Hi Chris, I was thinking about combobox field [like in evergreen](https://evergreen.segment.com/components/combobox) but came to the conclusion that it would bring no value as it is easily replaceable with custom select or autocomplete which are coming in next version (2.0.0). Can you provide some examples of usecases which are not covered by these components?

**Reply by cstrat** [reply]
Sorry for the very late reply!

I will wait to see how things work in 2.0 and will let you know.

Thanks again for all the great work. Loving using Mantine.

**Reply by croaky** [reply]
Hi! Mantine is looking great. I'm looking for a component similar to https://next.material-ui.com/components/autocomplete/#multiple-values and wondering if the intent of this issue was the same. I see Autocomplete in 2.0 but wasn't sure from the docs if it was possible to have multiple selected options, maybe displayed as "pills" with "x"s to remove each selection?

**Reply by rtivital** [reply]
Hi Dan, currently we do not have such component right now, it will be added in next minor release

**Reply by croaky** [reply]
Awesome, thank you!
